app/constants/tasks.js

```diff
@@ -1,8 +1,64 @@
-const STATUS = {
-  ACTIVE: 'active',
-  BLOCKED: 'blocked',
-  COMPLETED: 'completed',
-  PENDING: 'pending',
-};
+const TASK_STATUSES = [
```

Comment:
I wonder if this should be a map, instead of an array.

If I want to look up if there's a key called `TASK_STATUSES.<keyname>`, we have no easy way of doing it anymore. What do we benefit from making this an array here?

Reply:
we're using this array to show the sidebar options, if we use map, the order would be messed up, right?

Reply:
@swarajpure We can take the enum list, and make it into an array using [Object.keys](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys)

Reply:
My concern with using an object is, will we be able to ensure the order of the keys? If the order is messed up, the list will obtained from `Object.keys` will also have different order and the sidebar won't have predictable order. Someone might say "I remember `In Progress` was the third option last week, why and how is it changed now?" @ankushdharkar

Reply:
@swarajpure Good point. So we can do something like this:
```js
const DISPLAY_LIST = [ IN_PROGRESS, COMPLETED, UNASSIGNED ];
```

Why will this order change?